ui/apiserver/apiserver.go

```diff
@@ -26,6 +26,7 @@ const (
 )
 
 const PreCheckEndpointURL = "https://merlinite-7b-vllm-openai.apps.fmaas-backend.fmaas.res.ibm.com/v1"
+const InstructLabBotUrl = "http://bot:8081"
```

Comment:
this is different from the above: `http://instructlab-bot:8081`, is that important?

Reply:
This is default value for our dev environment. In prod, we use name the bot instructlab-bot, so that requires override in case of ansible.